firedrake/mg/interface.py

```diff
@@ -8,6 +8,9 @@
 __all__ = ["prolong", "restrict", "inject"]
 
 
+native = frozenset(["Lagrange", "Discontinuous Lagrange", "Real", "Q", "DQ"])
```

Comment:
I think I would prefer a single source of truth for this support. The more natural place is for it to be in embedded.py I think. To avoid circular import problems, you can do `from .embedded import native` at the top of the `check_arguments` function.

Reply:
Fixed in latest commit